extensions/amp-accordion/1.0/accordion.js

```diff
@@ -201,10 +212,21 @@ export function AccordionSection({
 
   return (
     <Comp {...rest} expanded={expanded} aria-expanded={String(expanded)}>
```

Comment:
Sorry I missed this -- `aria-expanded` should be on the header element (the same as the element which has `aria-controls`. 

Something also missing in 0.1 is that the content element should have `aria-labelledby` whose value is the `id` of its corresponding header element. Just calling out since this is good to have in mind when implementing so it can be cheaply supported later on, but again it's not crucial to implement for dev preview seeing as it's not in 0.1.

Reply:
@caroqliu to confirm. `aria-expanded` should be on the header, and ONLY on the header. Right?

@krdwan I these these two issues we need to address the right away, outside of `contentId` debate.

Reply:
1. `aria-expanded` - Updated to move it to header and only header.  I think this makes sense as header also has the `tabindex` attribute, so this is where screen readers will land for a11y purposes.  See this PR: https://github.com/ampproject/amphtml/pull/30721

2. `aria-labelledby` - I've tracked in this in the tracker issue https://github.com/ampproject/amphtml/issues/30445 `nice to have` since it's not in `0.1` and we also already have a relationship between `content` and `header` via `aria-controls` on `header` and `id` on `content`.  Please let me know if you disagree!

@caroqliu @dvoytenko

Reply:
Yes @dvoytenko that's right, I'll review the new changes locally with SR to confirm :)

Reply:
Thanks!!